goty-server/src/main/kotlin/com/aleinin/goty/result/ResultResponse.kt

```diff
@@ -0,0 +1,8 @@
+package com.aleinin.goty.result
+
+data class ResultResponse(
+    val gamesOfTheYear: List<ScoredGameResult>,
+    val mostAnticipated: List<RankedGameResult>,
+    val bestOldGame: List<RankedGameResult>,
+    val participants: List<String>,
+)
```

Comment:
re-add val giveawayParticipants: List<String>

Reply:
Re-added in latest commit.